components/Table/Table.jsx

```diff
@@ -64,7 +68,16 @@ const Table = ({
           {page.map(row => {
             prepareRow(row);
             return (
-              <tr className="govuk-table__row" {...row.getRowProps()}>
+              <tr
+                className="govuk-table__row lbh-table__row--data"
+                {...row.getRowProps()}
+                onClick={() =>
+                  Router.push(
+                    '/admin/applications/[clientGeneratedId]',
+                    `/admin/applications/${row.original.clientGeneratedId}`
```

Comment:
I had a look at the docs (https://nextjs.org/docs/api-reference/next/router#routerpush) and I can't see why we need the second argument. Please can you explain?

Reply:
```
Router.push(url, as, options)

url - The URL to navigate to. This is usually the name of a page
as - Optional decorator for the URL that will be shown in the browser. Defaults to url
```

In our case,  it's going to be:
```
Router.push('/admin/applications/[clientGeneratedId]', 'admin/applications/X7lKI3jHlNnbfdkahxHyC')
```

First param is defining the page (as in the file system), the second one is defining the actual route to navigate to. If we remove the first param we break the clientside navigation, so the browser will refresh on every push.

Reply:
OK, I think I understand. Thanks.

Reply:
Let me know if you want to know more 👍